2-Pricing/index.html

```diff
@@ -0,0 +1,428 @@
+                            1646 -71 859 -132 1582 -134 1606 l-6 42 1640 0 1640 0 -6 -42z"/>
+                            <path d="M1587 3299 c-25 -13 -45 -34 -58 -62 l-21 -43 76 -1234 c85 -1381 76
+                            -1299 155 -1340 51 -25 91 -25 142 0 27 14 46 34 60 63 l21 43 -76 1234 c-85
+                            1381 -76 1299 -155 1340 -50 25 -95 25 -144 -1z"/>
+                            <path d="M2488 3299 c-23 -12 -46 -35 -58 -59 -20 -39 -20 -57 -20 -1280 0
+                            -1224 0 -1241 20 -1280 23 -45 80 -80 130 -80 50 0 107 35 130 80 20 39 20 56
+                            20 1280 0 1224 0 1241 -20 1280 -37 73 -127 99 -202 59z"/>
+                            <path d="M3387 3299 c-77 -41 -68 37 -153 -1339 l-76 -1234 21 -43 c14 -29 33
+                            -49 60 -63 51 -25 91 -25 142 0 79 41 70 -41 155 1340 l76 1234 -21 43 c-37
+                            75 -127 103 -204 62z"/>
+                            </g>
+                        </svg>
+                        </button>
+                        <div class="checkout-row">
+                            <span>${item.name}</span> <span>$ ${monthlyPrices ? item.monthlyPrice : item.weeklyPrice }</span>
+                        </div>
+                    </div>
+                `
+                list.append(li)
+            })
+
+            const totalRow = document.createElement('li')
+            const totalPrice = cart
+                .map(item => (monthlyPrices) ? item.monthlyPrice : item.weeklyPrice)
+                .reduce((accum, iterator) => accum + iterator, 0)
+
+            totalRow.innerHTML = `
+                <div class="checkout-row total-row">
+                    <span>Total:</span> <span>$ ${totalPrice}</span>
+                </div>
+            `
+            list.append(totalRow)
+        }
+
+        const removeFromCart = function (event) {
+            const queryParams = new URLSearchParams(window.location.search)
+            const view = queryParams.get('view') === 'monthly'
+
+            const planToRM = event.currentTarget.dataset.subscription
+
+            const cart = JSON.parse(localStorage.getItem('cart'))
+            const idx = cart.findIndex(item => item.name == planToRM)
+            cart.splice(idx, 1)
+
+            localStorage.setItem('cart', JSON.stringify(cart))
+
+            populateCart(view)
+        }
+
+        togglePrices()
+
+        document.querySelector('#monthly-view')
+            .addEventListener('click', (e) => {
+            e.preventDefault()
+            history.pushState(null, '', '?view=monthly')
+            togglePrices()
+        })
+
+        document.querySelector('#weekly-view')
+            .addEventListener('click', (e) => {
+            e.preventDefault()
+            history.pushState(null, '', '?view=weekly')
+            togglePrices()
+        })
+ 
+        document.querySelectorAll('button.button')
```

Comment:
What does this query return? I think it is very general and a little dangerous if we added more buttons to the page in the future.

Reply:
Well spotted! It's actually selecting the "Add to cart" buttons in the cards. Initially `button` was a class with the generic styles for buttons, which at first were the "Add to Cart" buttons. I then created a `primary-btn` class but forgot to make this one more semantic.

Reply:
Perfect